docs/hardening_guide.md

```diff
@@ -0,0 +1,52 @@
+# Hardening Guide
+
+This document provides prescriptive guidance for hardening a production installation of RKE2. It outlines the configurations and controls required to address Kubernetes benchmark controls from the Center for Information Security (CIS).
+
+If RKE2 is run with the profile flag set to "cis-1.5", it performs a number of system checks and applies a restrictive NetworkPolicy and PodSecurityPolicy. 
+
+RKE2 checks for 5 sysctl kernel parameters (see below) as well as the existence of the "etcd" user. If any of the sysctl kernel parameters aren't set to the expected value or the "etcd" users doesn't exist, RKE2 will immediately cease operation. 
+
+The restrictive NetworkPolicy allows for only namespace traffic with the exception of DNS and applies to `kube-system`, `kube-public`, and `default` namespaces. The restrictive PodSecurityPolicy addresses CIS controls defined in section 5.2. More details can be found below.
+
+## Overview
+
+For more detail about evaluating a hardened cluster against the official CIS benchmark, refer to the [CIS Benchmark Rancher Self-Assessment Guide](cis_self_assessment.md).
+
+## Configure Kernel Runtime Parameters
+
+The following sysctl configuration is recommended for all nodes type in the cluster. Set the following parameters in `/etc/sysctl.d/90-kubelet.conf`.
```

Comment:
In one of the two install methods, i think the tar, I think @dweomer actually puts this file at a location somewhere in `/var/lib/rancher/rke2` ... so I think the instructions for this might be different for us. Actually, this is an inconsistency we might want to fix...need to talk to Jacob.

Reply:
It is the tar install method and its at `/usr/local/share/rke2/rke2-cis-sysctl.conf`

Reply:
Yes. An instruction to copy the sysctl content into `/etc/sysctl.d` should be sufficient.